migrations/20211129141831_cleanup_public_key.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE public_key
+	DROP COLUMN signing_public_key,
+	DROP COLUMN signing_public_key_type;
```

Comment:
There was thoughts to use the signing public key for authorization, but this is no longer needed.